user-docs/modules/ROOT/pages/booting-the-simplified-installer.adoc

```diff
@@ -0,0 +1,147 @@
+include::_attributes.adoc[]
+
+= Boot the Simplified Installer
```

Comment:
Are we sticking with "simplified installer" or should we start using "simplified provisioner"?

@pcdubs @nullr0ute

Reply:
The later, so "simplified provisioner"

Reply:
done